recipes/libcurl/all/conanfile.py

```diff
@@ -303,7 +303,7 @@ def _patch_autotools(self):
                                   "AC_CHECK_LIB(z,",
                                   f"AC_CHECK_LIB({zlib_name},")
             replace_in_file(self, configure_ac,
-                                  "-lz ",
+                                  "-lz",
```

Comment:
This change has a good chance to break `with_zstd=True`. There was a good reason for this extra space in pattern.

Reply:
Is there an upstream proposal to add the configure script option?